build/RELEASING.md

```diff
@@ -0,0 +1,19 @@
+﻿# Release process

```

Comment:
This is just notes for now once we automated the process we can update this file